local-storage-service/server/Dockerfile

```diff
@@ -0,0 +1,6 @@
+FROM openjdk:11
+VOLUME /tmp
+ADD target/local-storage-service-server.jar local-storage-service-server.jar
+ENV FILE_STORAGE_DIR_PATH "/opt/verapdf/localstorageservice/files"
```

Comment:
Please check where data is stored in typical DB container and use the similar pattern. 
I assume VOLUME should point to the same directory.